src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
 
+        List<String> dataFromFile = readFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeFile(report, toFileName);
+    }
+
+    private List<String> readFile(String fromFileName) {
+        List<String> list = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                list.add(value);
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        return list;
+    }
+
+    private String createReport(List<String> dataFromFile) {
+        final StringBuilder builder = new StringBuilder();
+        final int IndexOfName = 0;
```

Comment:
Нужно вынести эту константу в поле класса. 
Нужно назвать эту константу в соответствии с правилами именования констант
```suggestion
        private static final int INDEX_OF_NAME = 0;
```